packages/client/validation/makeUpdateAgendaItemSchema.ts

```diff
@@ -4,10 +4,10 @@ import legitify from './legitify'
 export default function makeUpdateAgendaItemSchema() {
   return legitify({
     id: compositeId,
-    content: (value) => value.trim().max(63, 'Try something a little shorter'),
-    isActive: (value) => value.boolean(),
-    pinned: (value) => value.boolean(),
-    sortOrder: (value) => value.float(),
+    content: (value: any) => value.trim().max(63, 'Try something a little shorter'),
```

Comment:
-1 The type is `value: Legitity` here.

We can also type the function `legitify` and then TS infers the type
```
type ExpectedLegitity = (value: Legitity) => Legitity

const legitify = (expected: {[key: string]: ExpectedLegitity}) => (actual: any) => {
 ```